README.md

```diff
@@ -44,35 +45,57 @@ Note: There are different HTTP methods other than GET. HTTP (Hypertext Transfer
 **DELETE:** deletes data.
 
 #### JSON
+
 JSON is a lightweight format that allows data to be transferred across the web by using key-value pairs. When we send information between applications it needs to be packaged up, and unpackaged when it is received. We do this using `JSON.stringify()` to package and `JSON.parse()` to unpackage.
 
 ### Exercise: write a fetch request that lists your repos
 
 Let's inspect the `response` object we receive in more depth:
 
 - Type/copy the following into the console. Remember to replace `<githubhandle>` with yours:
+
 ```js
 fetch('https://api.github.com/users/<githubhandle>/repos')
   .then(function(data) {
     console.log(data);
   })
   .catch(function(error) {
     console.log(error);
-  })
+  });
 ```
+
 - Look at the object that comes back. Notice:
   - `Response.body`: the readable stream of the response's body.
   - `Response.headers`: HTTP headers allow the client and the server to pass additional information with the request or the response. An example are status codes.
 
 Status codes tell us if a request was successful. The main groups are:
+
 - 2: Success codes. e.g. `200`
 - 3: Redirection e.g. `302`
 - 4: Error, Unfound e.g. `404`
 - 5: Server Error e.g. `500`
 
-### Fix a broken API request
+### Exercise: Use Open Movie Database API to fetch Harry Potter films and make a poster gallery.
+
+1. Make an API call to `http://omdbapi.com/<querystring>` in `poster-gallery/script.js`. Look at the documentation to work out which url to use. You should expect an array of 10 objects with a title, year, imdbID, Type, and Poster property e.g.
 
```

Comment:
suggested rephrase:
1. make an API call to OMDb movies to get all the information they have on Harry Potter. Read the docs [here](http://omdbapi.com/) to try and work out the url you need to use to make the API call